common/common.go

```diff
@@ -17,4 +17,6 @@ var (
 func Run(cfg *TestConfig) {
 	runBasic()
 	runDNS(cfg.BastionSupported)
+	runStorage()
+	cleanupStorage()
```

Comment:
This won't guarantee the order these are run.

Reply:
Removed the cleanupStorage 👍🏻